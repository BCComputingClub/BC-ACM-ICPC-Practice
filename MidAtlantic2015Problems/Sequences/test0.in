1 2 -1 4
2 4 8 -1
7 8 -1 21
5 -1 11 14
-1 2 4 6
-1 -1 -1 -1
