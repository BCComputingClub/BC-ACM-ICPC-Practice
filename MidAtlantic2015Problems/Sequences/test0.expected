3
16
-1
8
-1
